lib/postgrex.ex

```diff
@@ -96,6 +107,21 @@ defmodule Postgrex do
       from the ssl docs;
     * `:socket_options` - Options to be given to the underlying socket
       (applies to both TCP and UNIX sockets);
+    * `:idle_interval` - Ping connections after a period of inactivity in milliseconds.
+      Defaults to 1000ms;
+    * `:target_server_type` - Allows opening connections to only a server with the required state.
+      The allowed values are `:any`, `:primary` and `:secondary` (default: `:any`);
```

Comment:
```suggestion
    * `:target_server_type` - Allows opening connections to only a server in the given
      replica mode. The allowed values are `:any`, `:primary` and `:secondary`
      (default: `:any`);
```

WDYT?

Reply:
I like it! Indeed, it clarifies well the interaction between the two new options.